care/facility/models/json_schema/asset.py

```diff
@@ -1,6 +1,6 @@
-CAMERA_DETAILS = {
+META = {
     "$schema": f"http://json-schema.org/draft-07/schema#",
     "type": "object",
-    "properties": {"url": {"type": "string"}, "preset": {"type": "number"}},
+    "properties": {"camera": {"url": {"type": "string"}, "preset": {"type": "number"}}},
```

Comment:
JSON schema has an option to keep the camera key as optional, mark the camera field as optional.

Reply:
The latest draft of JSON schema only seems to have `required` - https://json-schema.org/draft/2020-12/json-schema-validation.html#rfc.section.6.5.3

Aren't non-required keys optional by default?

Reply:
My bad! I thought it was the other way around.